packages/react/src/domain/handles.ts

```diff
@@ -29,6 +30,10 @@ const SSO_FACTORS = ["oidc", "saml"];
 function getPossibleHandleTypes(factor: Factor): Set<HandleType> {
```

Comment:
Should we change the signature of this function? Looks like it can receive types other than `Factor`?